JitsiConference.js

```diff
@@ -718,6 +722,20 @@ JitsiConference.prototype._getMediaSessions = function() {
     return sessions;
 };
 
+/**
+ * Registers event listeners on the RTC instance.
+ * @param {RTC} rtc - the RTC module instance used by this conference.
+ * @private
+ * @returns {void}
+ */
+JitsiConference.prototype._registerRtcListeners = function(rtc) {
```

Comment:
There is already a listener registered for RTCEvents.DATA_CHANNEL_OPEN event here - https://github.com/jitsi/lib-jitsi-meet/blob/master/JitsiConferenceEventManager.js#L532. Can we add this functionality there instead of adding another listener here ?

Reply:
Yeah I was thinking exactly about that, but JitsiConferenceEventManager feels more like an event forwarder the way I understand it. I didn't want to add more complex logic there, especially one related to the streams. Let me know if that make sense, otherwise I will move it there.

Reply:
It does make sense to have it here then. Is there a reason why you chose not to do it the way its done for legacy vidyoType in RTC.js and add the listener here instead ?

Reply:
The reason for doing it here was the fact that `_sendBridgeVideoTypeMessage` is in the `JitsiConference`. In the new flow the video type is not stored in the RTC module, but taken from the local tracks state. Maybe eventually it could be moved to the RTC module, but looking at the existing usages of `_sendBridgeVideoTypeMessage` it's not a simple change.